src/main/java/controller/BetHandler.java

```diff
@@ -3,9 +3,11 @@
 import dao.BetDao;
 import dao.impl.BetDaoImpl;
 import java.util.Scanner;
+import lib.Inject;
 import model.Bet;
 
 public class BetHandler implements ConsoleHandler {
+    @Inject
     private BetDao betDao = new BetDaoImpl();
```

Comment:
```suggestion
    private BetDao betDao;
```
If we use Injector, we don't need to write init fields with "new" keyword, that's the whole point